src/draw/sw/lv_draw_sw.h

```diff
@@ -162,7 +162,7 @@ void lv_draw_sw_transform(lv_draw_unit_t * draw_unit, const lv_area_t * dest_are
                           int32_t src_w, int32_t src_h, int32_t src_stride,
                           const lv_draw_image_dsc_t * draw_dsc, const lv_draw_image_sup_t * sup, lv_color_format_t cf, void * dest_buf);
 
-#if LV_USE_VECTOR_GRAPHIC
+#if LV_USE_VECTOR_GRAPHIC && (LV_USE_THORVG_EXTERNAL || LV_USE_THORVG_INTERNAL)
```

Comment:
Should we use LV_USE_THORVG instead.

Reply:
I think for now, there is no need to introduce new macros.

Reply:
I agree, `(LV_USE_THORVG_EXTERNAL || LV_USE_THORVG_INTERNAL)` is used only on a few places

Reply:
Let's go in this way. 
LV_USE_LZ4_INTERNAL/EXTERNAL has similar problem, but I have defined LV_USE_LZ4 too.

Reply:
I would remove `LV_USE_LZ4` too to make the user set only 1 config option.